src/Header/SetCookie.php

```diff
@@ -378,7 +378,7 @@ public function getMaxAge()
      */
     public function setExpires($expires)
     {
-        if ($expires === null) {
+        if ($expires === null || $expires === '') {
```

Comment:
No, it seems `0` is not allowed. See: https://github.com/zendframework/zf2/issues/4684#issuecomment-19951416